docs/docs/release-notes/0.1.2.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: doc-page
+title: 0.1.2 release notes
+---
+
+0.1.2 will be the first public release of Dotty.
+Being first, those notes are incomplete.
+This document is work in progress until 0.1.2-final is released.
+This note is up-to-date based on 0.1.2-RC1 and will be updated 
+with changes that happen during RC stabilization cycle.
+
+# Required Java Version
+  
+Dotty 0.1.2 targets Java 8. We don't have plans so far to add support for earlier versions of Java. 
+
+# IDE support for Dotty
+
+# Reporting Bugs / Known Issues
+
+Please [file](https://github.com/lampepfl/dotty/issues) any bugs you encounter. If you’re unsure whether something is a bug, 
+please ask on Dotty [gitter channel](https://github.com/lampepfl/dotty).
+
+# SBT builds dotty that builds dotty
+
+
+# Dotty Doc
+
+Dotty has added support for Dotty Doc.
+Actually, the dotty documentation site [http://dotty.epfl.ch/docs/](http://dotty.epfl.ch/docs/) is generated by it.
+Looks nice, doesn't it?
+
+#Other implemented features:
+
+This release ships with the following features:
+
+ - TASTY support by [@odersky], [@xeno-by] and [@darkdimius] [\[1\]][1]
+ - HK-types support by [@odersky], [@smarter] and [@darkdimius] [\[2\]][2]
+ - Intersection and Union Types by [@odersky] [\[3\]][3] [\[4\]][3]
+ - Enumerations by [@odersky] [\[4\]][4] [\[5\]][5] [\[6\]][6]
+ - Implicit By-Name Parameters by [@odersky] [\[7\]][7]
+ - Miniphases-based design by [@darkdimius], [@odersky] and [@olhotak] [\[8\]][8]
+ - @static methods support by [@darkdmius] [\[9\]][9]
+ - Non-blocking lazy vals by [@darkdimius] [\[10\]][10]
+ - JVM code emission by [@magarciaEPFL] and [@darkdimius] [\[11\]][11]
+ - efficient multi-dimensional array allocation by [@darkdimius] [\[12\]][12]
+ - default-method based trait composition [@odersky] and [@darkdimius] [\[13\]][13] [\[14\]][14]
+ - Trait parameters by [@odersky] [\[15\]][15]
+ - Working contravariant implicits by [@odersky] [\[16\]][16]
+ - Option-less pattern matching by [@odersky], [@darkdimius] and [@OlivierBlanvillain] [\[17\]][17]
+ - SAM-based anonymous functions by [@magarciaEPFL], [@darkdimius] and [@retronym] [\[18\]][18]
+ - Patter matching support by [@darkdimius] [\[19\]][19]
+ - Value classes support by [@smarter] and [@darkdimius] [\[20\]][20]
+ - Pattern matching exhaustively checks by [@liufengyun] [\[21\]][21]
+ - Tailrec optimization by [@darkdimius] [\[22\]][22] [\[23\]][23]
+ - VS code protocol implementation by [@smarter] [\[24\]][24]
+ - VS code plugin by [@smarter] [\[24\]][24]
+ - Vulpix: new test-suite by [@felixmulder] that replaces partest by [@vsalvis] [\[25\]][25]
```

Comment:
new test-suite <ins>driver</ins>? Or something like that. It reads as if you have replaced the tests of the test suite.